en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md

```diff
@@ -99,52 +111,46 @@ Follow the steps given below to create the data service file:
     | URL                                | jdbc:mysql://localhost:3306/Employees |
     | User Name                          | root                                  |
 
-5.  Save the data service.
+4.  Click **Test Connection** to expand the section.
+    ![](../../assets/img/tutorials/data_services/test_connection.png)
 
-A data service file (DBS file) will now be created in your data service
-project. Shown below is the project directory.
+5.  Click the **Test Connection** button to verify the connectivity between the MySQL datasource and the data service.
+6.  Save the data service.
 
-![](../../assets/img/tutorials/data_services/119130577/119130593.png)
-
-#### Creating a query
+##### Creating a query
 
 Let's write an SQL query to GET data from the MySQL datasource that you
 configured in the previous step:
 
-1.  Select the data service you created in the previous step.
-2.  Right-click and click **Add Query** .  
-    ![](../../assets/img/tutorials/data_services/119130577/119130591.png)
+1.  Click **Queries** to expand the section. 
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
+2.  Click **Add New** to open the **Add Query** page.
+    ![](../../assets/img/tutorials/data_services/add_query.png)
```

Comment:
@DMadhuranga - Shall we add a screenshot for this? Same as for 'datasource connection details'.
Let's keep this pattern of screenshots consistent.